lib/harvesting/errors.rb

```diff
@@ -4,4 +4,7 @@ class AuthenticationError < StandardError
 
   class UnprocessableRequest < StandardError
   end
+
+  class ExceedRateLimit < StandardError
```

Comment:
Could you rename this class as `RateLimitExceeded`?

Reply:
updated 2c2a2d2bd3a69620e1dee3e49809626e6052b15a